doc/releases/reef.rst

```diff
@@ -73,15 +73,15 @@ Notable Changes
   The user might temporarily mute this warning using
   ``ceph health mute POOL_APP_NOT_ENABLED``.
 
-* An overview page for rgw to show the overall status of rgw components
+* Dashboard: An overview page for rgw to show the overall status of rgw components
 
-* Added management support for RGW Multi-site and Ceph FS Subvolumes and groups
+* Dashboard: Added management support for RGW Multi-site and Ceph FS Subvolumes and groups
```

Comment:
Since @anthonyeleven left some nits, I'll do too ;)  CephFS (i.e. no space in between) and it would be nice to end these with periods.